RuntimeError: module compiled against API version 0xf but this version of numpy is 0xe
  0%|          | 0/100 [00:00<?, ?it/s]  1%|          | 1/100 [00:16<27:12, 16.49s/it]  6%|▌         | 6/100 [00:21<04:39,  2.97s/it] 43%|████▎     | 43/100 [00:22<00:16,  3.36it/s]100%|██████████| 100/100 [00:22<00:00,  4.49it/s]
Exception ignored in: <function Pool.__del__ at 0x2b156aee1160>
Traceback (most recent call last):
  File "/home/n9455647/.conda/envs/conda_test/lib/python3.8/multiprocessing/pool.py", line 268, in __del__
  File "/home/n9455647/.conda/envs/conda_test/lib/python3.8/multiprocessing/queues.py", line 362, in put
AttributeError: 'NoneType' object has no attribute 'dumps'
